### PR TITLE
MERGE BLOCKED: Chore/change profile to account

### LINKED ISSUE #71: [merge blocked by spark] Rename "Profile" page to "Account" page
- [x] Search for text references to Profile page and replace with references  to Account page. 
- [x] Change urls in links from /profile to /account
- [x] DO NOT MERGE until Spark team executes change on their side.

### PR DESCRIPTION
**Pull request by johnnymatthews**
The _Profile_ page is now called _Account_. This PR updates all occurrences of `Profile` and `profile` to `Account` and `account`

Closes #71.

### CODE REVIEW

File: docs/how-tos/store.md [hunk 1]
```diff
@@ -159,7 +159,6 @@ You can also get more information about the status of your data. See the [query
 [concepts-decentralized-storage]: ../concepts/decentralized-storage.md
 
 <!-- links to the web3.storage site -->
-[site-profile]: https://web3.storage/profile/
```
Comment (johnnymatthews) [inline docs/how-tos/store.md:162]: This link is not used within this page.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by terichadbourne** [comment]
@johnnymatthews I think you missed a page. Can you please search the full repo to ensure you've also caught the new generate an API token section

**Comment by johnnymatthews** [comment]
Yeahp, my grep skills clearly aren't great.

**Comment by johnnymatthews** [comment]
That should do it!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

### `docs/how-tos/generate-api-token.md`:
- The following link returned HTTP status code 404 (Not Found):
  https://web3.storage/account
- The following link returned HTTP status code 302 (Found):
  https://web3.storage/tokens
- Consider a shorter alternative to avoid wordiness.
  ```
  ...You'll need a free Web3.Storage account in order to generate an API token. I...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...o see an invitation to do this from the Getting started section of your acco...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ill be easy for you to remember later.  TIP You don't have to use the same A...
                                             ^^^
  ```
- Possibly missing comma found.
  ```
  ... to your clipboard. Keep your API token private Do not share your API token ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... clipboard. Keep your API token private Do not share your API token with any...
                                             ^^
  ```
- It seems that “to” is missing before the verb.
  ```
  ...emove it. Deleting a token is permanent Remember that if you delete an API t...
                                             ^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...nd line. If you haven't yet explored in depth how to store data using Web3.S...
                                             ^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```
- Consider using a more formal alternative to avoid the overused word “more”.
  ```
  ...se the JavaScript client library to get more information about the status of...
                                             ^^^^^^^^^^^^^^^^
  ```

**Comment by jessicaschilling** [comment]
@terichadbourne @johnnymatthews Please don't merge yet. We need to change screenshots as well. I'll take care of that.

**Comment by jessicaschilling** [comment]
@terichadbourne @johnnymatthews Actually, strike that. It's better if I just do a separate PR replacing every screenshot. Feel free to merge.

**Review by jessicaschilling — APPROVED** [review]

**Comment by terichadbourne** [comment]
@jessicaschilling We **can't merge this regardless** because the Account page doesn't exist yet - just qeueing it up. If the screenshots you're referring to are for the Profile/Account page you can put them here to trigger at the same time. You could hack the change for the screenshots in an inspector or wait until the change is actually made, whichever you prefer. 

@rafaelramalho19 can you please let us know once Profile has changed to Account on main site so we don't pull the trigger prematurely?

**Review by terichadbourne — CHANGES_REQUESTED** [review]
Just working the system to try to prevent us from accidentally merging before the Account page exists.

**Comment by jessicaschilling** [comment]
@terichadbourne I'll repurpose https://github.com/web3-storage/docs/issues/86 to an "update all the screenshots" issue and psd-bash accordingly.

**Comment by filecorgi** [comment]
- I optimized some images for you! See the commit with the comment `Automatically optimized images [ci skip].` in this PR for details.
- Vuepress build was successful!

### `docs/how-tos/generate-api-token.md`:
- The following link returned HTTP status code 404 (Not Found):
  https://web3.storage/account
- The following link returned HTTP status code 302 (Found):
  https://web3.storage/tokens
- Consider a shorter alternative to avoid wordiness.
  ```
  ...You'll need a free Web3.Storage account in order to generate an API token. I...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...o see an invitation to do this from the Getting started section of your acco...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ill be easy for you to remember later.  TIP You don't have to use the same A...
                                             ^^^
  ```
- Possibly missing comma found.
  ```
  ... to your clipboard. Keep your API token private Do not share your API token ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... clipboard. Keep your API token private Do not share your API token with any...
                                             ^^
  ```
- It seems that “to” is missing before the verb.
  ```
  ...emove it. Deleting a token is permanent Remember that if you delete an API t...
                                             ^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...nd line. If you haven't yet explored in depth how to store data using Web3.S...
                                             ^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```
- Consider using a more formal alternative to avoid the overused word “more”.
  ```
  ...se the JavaScript client library to get more information about the status of...
                                             ^^^^^^^^^^^^^^^^
  ```

**Comment by alanshaw** [comment]
> Just working the system to try to prevent us from accidentally merging before the Account page exists.

It exists now!

**Comment by jessicaschilling** [comment]
@terichadbourne per https://github.com/web3-storage/docs/pull/102#issuecomment-889078060 , merging 😊

**Pull request merged**